the test depends on a pem from a csr which fails with

Error adding request extensions from section cert_ext
689CF0F7FF7F0000:error:11000079:X509 V3 routines:v2i_AUTHORITY_KEYID:no issuer certificate:crypto/x509/v3_akid.c:156:
689CF0F7FF7F0000:error:11000080:X509 V3 routines:X509V3_EXT_nconf_int:error in extension:crypto/x509/v3_conf.c:48:section=cert_ext, name=authorityKeyIdentifier, value=keyid:always,issuer:always
gmake[3]: *** [Makefile:3984: unit/cert-client.csr] Error 1

this is probably because with openssl 3.2 csr's default to x609v3 which seems to
not like the config.
there are no docs how to make it v1 again (only docs for how to make it v3 with
older openssl which defaulted to v1..)
--- a/Makefile.am
+++ b/Makefile.am
@@ -426,7 +426,7 @@
 unit_tests += unit/test-cmac-aes \
 		unit/test-hmac-md5 unit/test-hmac-sha1 unit/test-hmac-sha256 \
 		unit/test-prf-sha1 unit/test-kdf-sha256 \
-		unit/test-crypto unit/test-eapol unit/test-mpdu \
+		unit/test-crypto unit/test-mpdu \
 		unit/test-ie unit/test-util unit/test-ssid-security \
 		unit/test-arc4 unit/test-wsc unit/test-eap-mschapv2 \
 		unit/test-eap-sim unit/test-sae unit/test-p2p unit/test-band \
@@ -501,28 +501,6 @@
 				src/ie.h src/ie.c
 unit_test_mpdu_LDADD = $(ell_ldadd)
 
-unit_test_eapol_SOURCES = unit/test-eapol.c \
-				src/crypto.h src/crypto.c \
-				src/ie.h src/ie.c \
-				src/watchlist.h src/watchlist.c \
-				src/eapol.h src/eapol.c \
-				src/eapolutil.h src/eapolutil.c \
-				src/handshake.h src/handshake.c \
-				src/eap.h src/eap.c src/eap-private.h \
-				src/eap-tls.c src/eap-ttls.c \
-				src/eap-md5.c src/util.c \
-				src/eap-tls-common.h src/eap-tls-common.c \
-				src/erp.h src/erp.c \
-				src/band.h src/band.c \
-				src/mschaputil.h src/mschaputil.c
-unit_test_eapol_LDADD = $(ell_ldadd)
-unit_test_eapol_DEPENDENCIES = $(ell_dependencies) \
-				unit/cert-server.pem \
-				unit/cert-server-key-pkcs8.pem \
-				unit/cert-client.pem \
-				unit/cert-client-key-pkcs8.pem \
-				unit/tls-settings.8021x
-
 unit_test_util_SOURCES = src/util.h src/util.c src/band.c src/band.h \
 				unit/test-util.c
 unit_test_util_LDADD = $(ell_ldadd)
